.github/dependabot.yml

```diff
@@ -6,4 +6,3 @@ updates:
       interval: "weekly"
     ignore:
       - dependency-name: "@formatjs/intl-pluralrules"
```

Comment:
I tried to upgrade this as well but ran into all kinds of breaking changes and ESM issues. It's not a required change, so delaying that for now.